neps/nep-0509.md

```diff
@@ -376,36 +376,38 @@ After that, for each height in the epoch, [EpochInfo::sample_chunk_validators](h
 
 This way, everyone tracking block headers can compute chunk validator assignment for each height and shard. 
 
-### Limits
+### Size limits
 
 `ChunkStateWitness` is relatively large message. Given large number of receivers as well, its size must be strictly limited.
 If `ChunkStateWitness` for some state transition gets so uncontrollably large that it never can be handled by majority of validators, then its shard gets stuck.
 
-All the limits are described [here](https://github.com/near/nearcore/blob/b34db1e2281fbfe1d99a36b4a90df3fc7f5d00cb/docs/misc/state_witness_size_limits.md).
-Additionally, we have limit on currently stored chunk endorsements, because malicious chunk validators can spam these as well.
+We try to limit the size of the `ChunkStateWitness` to 16 MiB. All the limits are described [in this section](https://github.com/near/nearcore/blob/b34db1e2281fbfe1d99a36b4a90df3fc7f5d00cb/docs/misc/state_witness_size_limits.md).
+Additionally, we have limit on currently stored partial state witnesses and chunk endorsements, because malicious chunk validators can spam these as well.
+
+### Partial state witness
```

Comment:
This looks like an empty header, is it supposed to be here?

Reply:
Right! Thanks! :)